org.idempiere.test/src/org/idempiere/test/AbstractTestCase.java

```diff
@@ -226,6 +226,5 @@ protected String getTrxName() {
 	 */
 	static void shutdown() {
 		Adempiere.stop();
```

Comment:
I think this can be removed too, seems just a waste of cpu time that we are calling startup for each test class.

Reply:
Done.
In my eclipse tests the time spent decreased from 3:23 to 3:07